File: Python/uberBlack.py
from car import Car
class UberBlack(Car):
    TypeCarAcceptrd= []
    SeatsMaterial=[]
    def __init__(self, license, Drive,TypeCarAcceptrd,SeatsMaterial):
        super().__init__(license, Drive)
        self.TypeCarAcceptrd=TypeCarAcceptrd
        self.SeatsMaterial=SeatsMaterial

File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;
class UberBlack extends Car {
     Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

  
    public UberBlack(String license, Account driver,Map<String, Map<String, Integer>> typeCarAccepte,ArrayList<String> seatsMaterial ){
        super(license, driver);
        this.typeCarAccepted=typeCarAccepte;
        this.seatsMaterial=seatsMaterial;
    }
    
}